packages/css/src/breadcrumbs.css

```diff
@@ -38,13 +38,21 @@
     }
   }
 
+  :dir(rtl) & li:where(:not(:last-child))::after {
+    rotate: 180deg;
+  }
+
   /* When link is direct child of Breadcrumbs, make it back button */
   & > :not(ol, ul)::before {
     background: currentColor;
     margin: 0;
     rotate: 180deg;
   }
 
+  :dir(rtl) & > :not(ol, ul)::before {
+    rotate: 0deg;
+  }
+
```

Comment:
```suggestion
  & > :not(ol, ul)::before {
    background: currentColor;
    margin: 0;
    rotate: 180deg;
    
    &:dir(rtl) {
      rotate: 0deg;
    }
  }

```
Would this make it a bit easier? Since we don't need to duplicate the selector.
`:dir` seems to work fine when nested like this

Reply:
No this makes the arrow flip the wrong way 
<img width="112" alt="bilde" src="https://github.com/user-attachments/assets/b9f3fa88-a4fa-4120-b2ab-34b9ab99e55f" />